### DISCUSSION TITLE
关于tx云网络诊断

### DISCUSSION THREAD

**Discussion by Zhongren233** [Mrs4s/go-cqhttp#1282] [Q&A]
为何修改dns为114后还是警告

本地连接聊天服务器丢包率为 100%, 可能出现消息丢失/延迟或频繁掉线等情况, 请检查本 地网络状态.
![image](https://user-images.githubusercontent.com/62706664/146960634-d808753a-33f0-4c06-b306-0e2d24b99c10.png)

![image](https://user-images.githubusercontent.com/62706664/146960548-04f89f67-8562-4483-ab91-cb7e9f56c7fd.png)

**Comment by Zhongren233** [comment]
关联issue https://github.com/Mrs4s/go-cqhttp/issues/1317

**Comment by oicv** [comment]
使用ROOT权限、管理员权限启动进程就没问题了

**Reply by Zhongren233** [reply]
不够优雅 非要sudo一下
希望之后修复吧

**Comment by DaBoSAMA** [comment]
老哥，你腾讯云是怎么修改DNS 的
